db/database_test.go

```diff
@@ -674,28 +674,28 @@ func TestAllDocsOnly(t *testing.T) {
 	assert.Equal(t, 100, len(changes))
 
 	for i, change := range changes {
-		seq := i + 1
+		seq := i
```

Comment:
Should change this to something like 'docIndex' so that it's clear that it's now unrelated to sequence.

Reply:
Sure Adam, this is committed to latest revision.